docs/howtoguides.rst

```diff
@@ -10,11 +10,14 @@ can be applied to a wider range of situations.
 They will help you to achieve a particular end result, but may require you to
 understand and adapt the steps to fit your specific requirements.
 
+The Pro Client comes pre-installed on every Ubuntu release so you do not need
+to install it.
+
```

Comment:
Consider tying this more explicitly with the sections below.

It does not add much standing alone and seems to say the same thing twice.

Could be rephrased to:

> The Pro Client comes pre-installed on every Ubuntu release, enabling you to attach machines to your Pro subscription and benefit from Pro services.

Alternatively, the next section could start:

> The Pro client comes pre-installed on every Ubuntu release.
> To access Ubuntu Pro services, you need to attach your machine [...]

This would make a more obvious link between pre-installation enabling attachment.

Reply:
I like the second one a lot - good suggestion :)